launching-software/infrastructure.md

```diff
@@ -100,7 +100,7 @@ Sandbox accounts - both cloud.gov and AWS - are available to all TTS staff for n
   authentication is in place. Exposing systems to the public without
   authentication requires an [ATO]({{site.baseurl}}/launching-software/lifecycle/#atos).
 - Sandbox accounts _must_ be used when you are sending internet traffic to a non-production system: tools such as `ngrok` and `localtunnel` are strictly forbidden since they can allow your laptop to be compromised.
-- No sensitive or [personally identifiable information (PII)](https://before-you-ship.18f.gov/privacy/) should be stored in sandbox accounts.
+- No sensitive or [personally identifiable information (PII)]({{site.baseurl}}/launching-software/privacy) should be stored in sandbox accounts.
```

Comment:
Don't forget the ending `/`, which is the canonical URL. This saves an extra request to redirect from no-slash to slash.

```suggestion
- No sensitive or [personally identifiable information (PII)]({{site.baseurl}}/launching-software/privacy/) should be stored in sandbox accounts.
```

Reply:
Oh interesting, I wonder why that is -- if you have any resources to understand this please send them my way.